solr/solr.go

```diff
@@ -100,6 +100,11 @@ type SolrMltResult struct {
 	Error          map[string]interface{}
 }
 
+// SolrGetResult is the parsed result for the RealTimeGetHandler response ie /get
+type SolrGetResult struct {
```

Comment:
To be consequent and more readable maybe we should call this SolrRealtimeGetResult?

Reply:
Yeah makes sense. I've made the changes